pkg/acrn/grub-hv.cfg

```diff
@@ -1,4 +1,4 @@
 # Change hypervisor to ACRN
 
 set_global hv_boot_cmd "multiboot --quirk-modules-after-kernel /boot/acrn.32.out uart=bdf@0:18.1"
-set_global hv_linux_cmd "module $linux Linux_bzImage"
+set_global hv_linux_cmd "module $linux Linux_bzImage root=/dev/mmcblk0p2 textrw rootwait console=tty0 console=ttyS0 earlyprintk=ttyS0 consoleblank=0 no_timer_check quiet loglevel=3 i915.nuclear_pageflip=1 i915.avail_planes_per_pipe=0x01010F i915.domain_plane_owners=0x011111110000 i915.enable_gvt=1 hvlog=2M@0x5de00000 memmap=0x200000\$0x5de00000 memmap=0x400000\$0x5da00000 ramoops.mem_address=0x5da00000 ramoops.mem_size=0x400000 ramoops.console_size=0x200000 reboot_panic=p,w module_blacklist=dwc3_pci i915.enable_guc=0x02 cma=64M@0-"
```

Comment:
Does it already work if you put $root here instead of hardwiring it to /dev/mmcblk0p2? Grub configuration should fill it up with the correct partition to boot.

Reply:
Good idea, let me try to use $root to replace /dev/mmclblk0p2 to see if it works. Will try it tomorrow.

Reply:
Veried with "root=$rootfs_root" and it works. I have updated the patches, please review.